packages/payer-logo/src/PayerLogo.js

```diff
@@ -94,9 +94,12 @@ const PayerLogo = ({ spaceId, payerId, clientId, ...props }) => {
 };
 
 PayerLogo.propTypes = {
+  /** String. Required. Client ID to use to fetch the payer's logo. */
+  clientId: PropTypes.string.isRequired,
```

Comment:
the type and required/optional should be redundant within the description so I'd remove that.

Reply:
done